.github/workflows/db-update.yml

```diff
@@ -24,99 +46,120 @@ on:
 
-    - name: Clear content of the database
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
-      uses: liquibase-github-actions/update@v4.23.0
-      with:
-        classpath: 'liquibase'
-        changeLogFile: 'population_prep_tables.xml'
-        username: ${{ secrets.DB_USER_NAME }}
-        password: ${{ secrets.DB_USER_PASSWORD }}
-        url: 'jdbc:postgresql://${{ env.DB_IP }}:5432/${{ inputs.DB_NAME }}'
+  db-content-update:
+    name: 'Database Content Update'
+    permissions: write-all
+    runs-on: ubuntu-latest
+    needs: db-schema-update
+#    if: ${{ github.event_name == 'repository_dispatch' || github.event_name == 'workflow_dispatch' }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
 
     - name: Setup python
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
       uses: actions/setup-python@v4
       with:
-        python-version: '3.9'
+        python-version: '3.10'
 
     - name: Update .env file
       run: |
         echo "PGUSER=${{ secrets.DB_USER_NAME }}" >> config/.env.local
         echo "POSTGRES_PASSWORD=${{ secrets.DB_USER_PASSWORD }}" >> config/.env.local
         echo "POSTGRES_DB=${{ inputs.DB_NAME }}" >> config/.env.local
         echo "POSTGRES_PORT=5432" >> config/.env.local
-        echo "POSTGRES_HOST=${{ env.DB_IP }}" >> config/.env.local
-        echo "FEEDS_DATABASE_URL=postgresql://${{ secrets.DB_USER_NAME }}:${{ secrets.DB_USER_PASSWORD }}@${{ env.DB_IP }}:5432/${{ inputs.DB_NAME }}" >> config/.env.local
-        echo "ENV=dev" >> config/.env.local
+        echo "POSTGRES_HOST=${{ inputs.DB_IP }}" >> config/.env.local
+        echo "FEEDS_DATABASE_URL=postgresql://${{ secrets.DB_USER_NAME }}:${{ secrets.DB_USER_PASSWORD }}@${{ inputs.DB_IP }}:5432/${{ inputs.DB_NAME }}" >> config/.env.local
+        echo "ENV=${{ inputs.ENVIRONMENT }}" >> config/.env.local
         cat config/.env.local
 
     - name: Install requirements and generate db model
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
       run: scripts/db-gen.sh
 
     - name: Download csv version of the database
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
       run: wget -O sources.csv https://bit.ly/catalogs-csv
 
-    - name: Validate file download
-      run: ls -la
-
     - name: Get full path of sources.csv
       id: getpath
       run: echo "PATH=$(realpath sources.csv)" >> $GITHUB_OUTPUT
 
-    - name: Populate Database
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
+    - name: Update Database Content
       run: scripts/populate-db.sh ${{ steps.getpath.outputs.PATH }} > populate.log
 
     - name: Upload log file for verification
-      if: inputs.RUN_POPULATE_SCRIPT == 'true'
       uses: actions/upload-artifact@v3
       with:
         name: populate.log
         path: populate.log
 
+  update-gcp-secret:
+    name: Update GCP Secrets
+#    if: ${{ github.event_name == 'repository_dispatch' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Do we need this conditional uncommented?

Reply:
Yes!